csrc/scheduler/normalization_utils.cpp

```diff
@@ -721,5 +728,620 @@ getOptionalInnerOuterPersistentBufferBatches(
   }
 }
 
+int64_t getAvailableSmemSize(
```

Comment:
Separated from `PersistentKernelScheduler::getPersistentBufferSize` used by runtime check of inner and innerOuter schedulers.